src/server/endpoints/covidcast_nowcast.py

```diff
@@ -20,9 +20,9 @@
 @bp.route("/", methods=("GET", "POST"))
 def handle():
     require_all(
-        "data_source", "time_type", "geo_type", "time_values", "signals", "sensor_names"
+        request, "data_source", "time_type", "geo_type", "time_values", "signals", "sensor_names"
```

Comment:
how about moving `request` to the line above to separate it from the long list of keys that follow, which all logically group together?

itd look like:
```
    require_all(request,
         "data_source", "time_type", "geo_type", "time_values", "signals", "sensor_names"
```